### PR TITLE
(#2) Synonym Table Component

### PR DESCRIPTION
**Pull request by karlikpj**

### CODE REVIEW

File: src/components/molecules/azlist/AZList.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.azlist_container .browse {
```
Comment (arcepaul) [inline src/components/molecules/azlist/AZList.scss:1]: regroup/nest azlist_container and contentzone styles

File: src/components/molecules/azlist/AZList.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	font-family: Montserrat, Avant Garde, Arial, sans-serif;
+	margin-left: 0;
+	margin-right: 1em;
+	display: inline-block;
+}
+.main-content a {
+	color: #2b7bba;
+}
+
+a {
```
Comment (arcepaul) [inline src/components/molecules/azlist/AZList.scss:27]: this needs to be scoped

File: src/components/molecules/definitionitem/DefinitionItem.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.results dt {
```
Comment (arcepaul) [inline src/components/molecules/definitionitem/DefinitionItem.scss:1]: these can be nested

**Review by arcepaul — CHANGES_REQUESTED** [review]

File: src/components/molecules/azlist/AZList.scss (hunk 1 above)
Comment (karlikpj) [reply src/components/molecules/azlist/AZList.scss:1]: so this is a mock component - was just a placeholder

File: src/components/molecules/azlist/AZList.scss (hunk 2 above)
Comment (karlikpj) [reply src/components/molecules/azlist/AZList.scss:27]: so this is a mock component - was just a placeholder

File: src/components/molecules/definitionitem/DefinitionItem.scss (hunk 3 above)
Comment (karlikpj) [reply src/components/molecules/definitionitem/DefinitionItem.scss:1]: so this is a mock component - was just a placeholder

File: src/components/atomic/Synonym/Synonym.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
```
Comment (arcepaul) [inline src/components/atomic/Synonym/Synonym.jsx:1]: filenames should be lowercase - synonym.jsx

File: src/components/atomic/Synonym/Synonym.jsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+import React from 'react';
+
+import './Synonym.scss';
```
Comment (arcepaul) [inline src/components/atomic/Synonym/Synonym.jsx:4]: lowercase

**Pull request closed without merging**